packages/opentelemetry-resources/src/platform/browser/detect-resources.ts

```diff
@@ -13,14 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import { Resource } from '../../Resource';
+import { detectResources as detect } from '../utils';
```

Comment:
Seems like an odd indirection to import and export this function? Also with the naming changes?

Reply:
fixed in the latest commit.